be/src/util/aes_util.cpp

```diff
@@ -113,10 +113,10 @@ int AesUtil::encrypt(AesMode mode, const unsigned char* source, uint32_t source_
     if (cipher == nullptr || (EVP_CIPHER_iv_length(cipher) > 0 && !iv)) {
         return AES_BAD_DATA;
     }
-    EVP_CIPHER_CTX_init(&aes_ctx);
+    EVP_CIPHER_CTX_reset(aes_ctx);
```

Comment:
Seems that `EVP_CIPHER_CTX_reset` is no longer needed here.

Reply:
I've checked the document of openssl about EVP_CIPHER_CTX_reset function. And got following explanations:
https://www.openssl.org/docs/man1.1.1/man3/EVP_CIPHER_CTX_reset.html

```
EVP_CIPHER_CTX_reset() clears all information from a cipher context and free up any allocated memory associate with it, except the ctx itself. 
This function should be called anytime ctx is to be reused for another EVP_CipherInit() / EVP_CipherUpdate() / EVP_CipherFinal() series of calls.
```

So, I think we can keep this function call as is.

Reply:
The init/clear is already done in `EVP_CIPHER_CTX_new`. And there's no reusing of `aes_ctx` here.